BigCommerce/Overview/01_General.md

```diff
@@ -0,0 +1,23 @@
+# General information
+
+*BigCommerce* is one of our partners in the MACH-Alliance. With its API-first approach to e-commerce, *BigCommerce* aims to simplify e-commerce on an enterprise level.   
+Through the integration of *BigCommerce* as an *Omni-Channel* extension, you will be able to connect your *BigCommerce* store(s) and thus manage all areas of work processes of online retailing centrally and from one place.
+
+The documentation refers to the *BigCommerce Integration* plugin version 1.1.0. There may be differences in the UI, features, and operation if your *BigCommerce Integration* plugin is installed in a different version.
+
```

Comment:
Heißt der Plugin eigentlich BigCommerce Integration? (Ich habe es bei mir in Sandbox nicht, daher kann ich nicht prüfen). Dann sollten wir die Liste in Docu Confluence dementsprechend korrigieren und den Namen des Ordners theoretisch auch ändern... Wir nehmen den Plugin Key als Name für den Ordner (Arbeitsbranch und in main nach dem Mergen). Ich vermute, die Webseite erkennt den Namen vom Plugin Key...

![image](https://github.com/Actindo-AG/Documentation/assets/99268392/ffe69fa2-8132-4791-a9bd-1d3314948697)

Reply:
ja